tools/image_analysis_support.py

```diff
@@ -0,0 +1,36 @@
+from scanomatic.generics import Model
+
+
+class VersionChangesModel(Model):
+    def __init__(self, **kwargs):
+        self.first_pass_change_1: float = 0.997
+        self.oldest_allow_fixture: float = 0.9991
+        super().__init__()
+
+
+def get_active_plates(
+    meta_data,
+    suppress_analysis,
+    graph_watch,
```

Comment:
I know this is dead code, but
```suggestion
    meta_data: dict,
    suppress_analysis: bool,
    graph_watch: Sequence,
```
perhaps?